pwd_guess.py

```diff
@@ -1362,11 +1396,14 @@ class ProbabilityCalculator(object):
     def __init__(self, guesser, prefixes=False):
         self.guesser = guesser
         self.ctable = CharacterTable.fromConfig(guesser.config)
-        #todo: decide which preprocessor to use here
-        self.preproc = ManyToManyPreprocessor(guesser.config)
+        self.config = guesser.config
+        if self.config.sequence_model == Sequence.MANY_TO_MANY:
```

Comment:
It probably makes sense to call the BasePreprocessor.fromConfig here rather than duplicating the logic.